docs/rules/max-len.md

```diff
@@ -1,71 +1,125 @@
-# Limit Maximum Length of Line (max-len)
+# enforce a maximum line length (max-len)
 
 Very long lines of code in any language can be difficult to read. In order to aid in readability and maintainability many coders have developed a convention to limit lines of code to X number of characters (traditionally 80 characters).
 
 ```js
-// max-len: ["error", 80, 4]; // maximum length of 80 characters
-var foo = { "bar": "This is a bar.", "baz": { "qux": "This is a qux" }, "difficult": "to read" }; // too long
+var foo = { "bar": "This is a bar.", "baz": { "qux": "This is a qux" }, "difficult": "to read" }; // very long
 ```
 
-
 ## Rule Details
 
-This rule is aimed at increasing code readability and maintainability by enforcing a line length convention. As such it will warn on lines that exceed the configured maximum.
+This rule enforces a maximum line length to increase code readability and maintainability.
 
 **Note:** This rule calculates the length of a line via code points, not characters. That means if you use a double-byte character in your code, it will count as 2 code points instead of 1, and 2 will be used to calculate line length. This is a technical limitation of JavaScript that is made easier with ES2015, and we will look to update this when ES2015 is available in Node.js.
 
-The following patterns are considered problems:
+## Options
+
+This rule has a number or object option:
+
+* `"code"` (default `80`) enforces a maximum line length
+* `"tabWidth"` (default `4`) enforces tabs to be treated as a specific width
```

Comment:
Recommend something like the following, but even better if you can simplify it :)

specifies the equivalent number of spaces for each tab character in the length calculation